### ISSUE TITLE
Ember build with sourcemaps fails

### ISSUE THREAD

**Issue by oquis** [damiencaselli/ember-cli-sentry#37]
I just updated to v2.1.1 and when I try to make a production build it fails, apparently because of raven-js and sourcemaps:

```
ember b -prod    
version: 1.13.8
Build failed.
File: assets/vendor.js
Invalid mapping: {"generated":{"line":68421,"column":-22232},"source":"bower_components/ember-data/ember-data.prod.js","original":{"line":1,"column":0},"name":null}
```

I know it says it failed with ember-data.prod.js, but if I remove raven-js from bower it works (using the 2.0.0 release)

**Comment by damiencaselli** [comment]
@oquis I have not encountered this issue, and it is not reproducible with a fresh application.

Can you try building using this repository : https://github.com/damiencaselli/ember-cli-sentry-demo?

I'm using `ember-cli@1.13.14`, though I doubt it makes any difference here.

**Comment by oquis** [comment]
@damiencaselli It still happens, but only when you enable sourcemaps. Here are the steps to reproduce:
- `git clone https://github.com/damiencaselli/ember-cli-sentry-demo.git`
- `cd ember-cli-sentry-demo.git`
- Add the following option to the app constructor in `ember-cli-build.js`:
```javascript
var app = new EmberApp(defaults, {
  // Add options here
  sourcemaps: {
    enabled: true,
    extensions: ['js']
  }
});
```
- Run `ember b -prod`

<img width="920" alt="screen shot 2016-01-19 at 10 33 32 a m" src="https://cloud.githubusercontent.com/assets/2367664/12425080/c7b5ed9c-be98-11e5-8245-be8ef1eda468.png">

**Comment by oquis** [comment]
I found the solution. For some reason the sourcemap can't be generated if importing the minified version of raven.js. To fix it, the non-minified version of raven.js has to be used. This line:
https://github.com/damiencaselli/ember-cli-sentry/blob/master/index.js#L10
has to be changed for `app.import(app.bowerDirectory + '/raven-js/dist/raven.js');`. Then the sourcemaps are generated with no problems. I tried to make a PR but don't have access :(
